src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -208,7 +229,7 @@ private EnumActionResult useHoe(ItemStack stack, EntityPlayer playerIn, World wo
         IBlockState iblockstate = worldIn.getBlockState(pos);
         Block block = iblockstate.getBlock();
         if (facing != EnumFacing.DOWN && worldIn.isAirBlock(pos.up())) {
-            if (block == Blocks.GRASS || block == Blocks.GRASS_PATH) {
+            if (block == Blocks.GRASS) {
```

Comment:
This allows creation of Farmland out of Grass if the Player clicks on dirt.
Should i remove it?

Reply:
Haven't looked at things yet but part of me is wondering if overall it should be the reconstructor behaves as it did before when right clicking grass, dirt, or grass paths... And then have it be when you are sneaking that it instead turns grass into grass paths.

Either way I think also adding the ability that right clicking coarse dirt turns it into regular dirt would make sense for this PR. (For reference right clicking coarse dirt with a regular hoe will turn it into regular dirt.)

Reply:
I think that's a bad idea as if you make path you most of the times want to be fast and you don't if you sneak.

Reply:
Actually thinking about it some more maybe a good middle ground would grass -> grass path, and then continue allowing the grass path -> farmland if they right click it again. Because if I wanted to turn grass into farmland I would get rather annoyed having to break it and place it again just to be able to convert it.

Reply:
For now let's go with the sneaking to make paths as pupnewfster said - I would think plenty of people already use the right click on grass to make farmland, just like a hoe does

Reply:
So i guess that if keeps being there and i add Grass Paths to the farmland if right?

Reply:
thiakil your comment just rendered after i replied, so i didn't replied to it but i think i will implement pupnewfster's new suggestion.

Reply:
normal right click -> pre-PR functionality; all pre-existing blocks
sneak+right click on grass -> grass path

Is how i understood it

Reply:
that was the first suggestion but i would realy prefer the second one.

Reply:
this is how i understood the second suggestion. is it okay like this or do you want me to change it to
"normal right click -> pre-PR functionality; all pre-existing blocks
sneak+right click on grass -> grass path"?

Reply:
I think the second suggestion (and how you changed things is fine for now). At some point we may want to give more configuration/upgrades to the atomic disassembler but for now this should be fine.